### ISSUE TITLE
Objetivo del sitio

### ISSUE THREAD

**Issue by zomars** [Maqueteros/bomberosculiacan#3]
# Objetivo principal del sitio

- Sin definir

# Objetivos secundarios

- Sin definir

# Propuestas de objetivos

- Recabar donaciones
- Reclutar voluntarios
- Dar a conocer información de contacto
- Procedimientos en casos de emergencias

**Comment by zomars** [comment]
Abrí este ticket para recabar ideas para el objetivo principal del sitio así como objetivos secundarios.

**Comment by Aldaz08** [comment]
Tambien creo que el objetivo principal debe ser el de las donaciones.

**Comment by marioecg** [comment]
Recabar donaciones y pienso que también sería importante fomentar el servicio a la comunidad.

**Comment by zomars** [comment]
@marioecg ¿Cómo se fomenta el servicio a la comunidad?

**Comment by marioecg** [comment]
pensándolo bien es más importante el objetivo de las donaciones @zomars

**Comment by angel8au** [comment]
Estoy de acuerdo en que las donaciones es el principal objetivo ademas del teléfono de contacto para emergencias. Como objetivos secundarios procedimientos de casos de emergencia, recaudar voluntarios. Así como educación con respecto a prevención de accidentes es otro objetivo que yo añadiría.

**Comment by zomars** [comment]
Para las donaciones se puede implementar Stripe: https://stripe.com/docs/accepting-donations

**Comment by zomars** [comment]
Incluso parece que pueden exentarlos de las comisiones: https://support.stripe.com/questions/fee-discount-for-nonprofit-organizations
